Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,9 +154,12 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
-        while line = read.gets
-          puts line if verbose
-          @output += line
+        while buf = read.read(8)
```

Comment:
I really don't think it's worth worrying about performance here given it's a blocking IO operation in a tight loop and only in our CI.